src/pages/library/datasetBuilder/DatasetBuilder.ts

```diff
@@ -281,15 +286,15 @@ export const CohortSelector = ({
   const [creatingCohort, setCreatingCohort] = useState(false);
 
   return h(Fragment, [
-    h(Selector, {
+    h(Selector as React.FC<SelectorProps<Cohort>>, {
```

Comment:
we should avoid using 'as' casting, since it can be more 'forgiving' then we like.  I recommend creating a const CohortSelector alias to capture this type info before the render block.  I believe other examples that Nick Watts can point you to do it this way.

Reply:
It turns out that `Selector` is a generic type so I fixed this by passing `Cohort` as the generic type argument.